docs/source/topics/support.rst

```diff
@@ -0,0 +1,101 @@
+Supported File Formats
+======================
+
+TARDIS
+------
+
+.. warning::
+
+    Import of TARDIS configuration files not supported yet, but will be
+    available in the future.
+
+HTCondor
+--------
+
+Job Imports
+~~~~~~~~~~~
+
+:term:`Jobs <Job>` can be created directly from HTCondor outputs. Via the
+``condor_history`` command from HTCondor, ClassAds describing a :term:`jobs <Job>`
+requested and used resources can be gathered and saved to a csv or json file.
+To sufficiently describe a :term:`job` for the simulation information about
+requested and used resources should be included in the export:
+
+requested resources:
+    RequestCpus, RequestWalltime, RequestMemory, RequestDisk
+
+used resources:
+    RemoteWallClockTime, MemoryUsage, DiskUsage_RAW, RemoteSysCpu, RemoteUserCpu
+
+additional job information:
+    QDate
+
+If csv is chosen as input file format every line represents a :term:`job`,
```

Comment:
Can you take another look at this paragraph and the 3 following ones? Some formulations read a bit jarring, e.g. "information about a jobs input files" has two plurals. Not sure if these are on purpose.

Reply:
Yes, this one is on purpose, meaning the "input files of a job". What do you propose to improve the reading?

Reply:
Sorry, this was outdated information that was still part of @tfesenbecker's commit. I adapted the information to our current plans.